src/pages/data.tsx

```diff
@@ -0,0 +1,280 @@
+export const slidesSetOne = [
+  {
+    url: "https://bellroy-cms-images.imgix.net/3954/slider-image01.jpg",
+    text: "Made from premium, enviormentally certified leather, that only gets better with age...",
+  },
+  {
+    url: "https://bellroy-cms-images.imgix.net/4413/slider-image02.jpg",
+    text: "and fewer of leather, to eliminate bulk from the start...",
+  },
+];
+
+const slidesSlidesSetTwo = [
+  {
+    url: "https://bellroy-cms-images.imgix.net/4414/slider-image03.jpg",
+    text: "their clever design features help keep them slim, even when they're full…",
+  },
+  {
+    url: "https://bellroy-cms-images.imgix.net/4425/slider-image04.jpg",
+    text: "so they'll be with you from Day One to Day 1000, and beyond.",
+  },
+];
+
+export const headerSubSections = [
+  {
+    text: "Bags",
+  },
+  {
+    text: "Wallets",
+  },
+  {
+    text: "Accessories",
+  },
+  {
+    text: "Tech",
+  },
+  {
+    text: "Travel",
+  },
+  {
+    text: "Collections",
+  },
+  {
+    text: "About Us",
+  },
+];
```

Comment:
I wouldn't have this here, this file seems more like data from fetched api and this is your way of mimicing it. This should go with your nav

Reply:
data moved into subsections

Reply:
Why is this data still available here?

Reply:
file deleted.